2014/12/01 05:19:08 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "/login".' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\Exportc-Corp\protected\views\layouts\main.php(87): CController->renderPartial('/login')
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#5 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#8 C:\xampp\htdocs\Exportc-Corp\protected\views\layouts\column1.php(6): CBaseController->endContent()
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#10 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(784): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#12 C:\xampp\htdocs\Exportc-Corp\protected\controllers\SiteController.php(32): CController->render('index')
#13 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#19 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#20 C:\xampp\htdocs\Exportc-Corp\index.php(13): CApplication->run()
#21 {main}
REQUEST_URI=/Exportc-Corp/
---
2014/12/01 05:19:17 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "site/login".' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\Exportc-Corp\protected\views\layouts\main.php(87): CController->renderPartial('site/login')
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#5 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#8 C:\xampp\htdocs\Exportc-Corp\protected\views\layouts\column1.php(6): CBaseController->endContent()
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#10 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(784): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#12 C:\xampp\htdocs\Exportc-Corp\protected\controllers\SiteController.php(32): CController->render('index')
#13 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#19 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#20 C:\xampp\htdocs\Exportc-Corp\index.php(13): CApplication->run()
#21 {main}
REQUEST_URI=/Exportc-Corp/
---
2014/12/04 04:25:16 [error] [exception.CException] exception 'CException' with message 'CAssetManager.basePath "C:\xampp\htdocs\Exportc-Corp\assets" is invalid. Please make sure the directory exists and is writable by the Web server process.' in C:\xampp\htdocs\yii\framework\web\CAssetManager.php:138
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CAssetManager.php(123): CAssetManager->setBasePath('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\yii\framework\web\CAssetManager.php(217): CAssetManager->getBasePath()
#2 C:\xampp\htdocs\yii\framework\web\CClientScript.php(510): CAssetManager->publish('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\yii\framework\web\CClientScript.php(547): CClientScript->getCoreScriptUrl()
#4 C:\xampp\htdocs\yii\framework\web\CClientScript.php(348): CClientScript->getPackageBaseUrl('jquery')
#5 C:\xampp\htdocs\yii\framework\web\CClientScript.php(208): CClientScript->renderCoreScripts()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(351): CClientScript->render('<!DOCTYPE html>...')
#7 C:\xampp\htdocs\yii\framework\web\CController.php(788): CController->processOutput('<!DOCTYPE html>...')
#8 C:\xampp\htdocs\Exportc-Corp\protected\controllers\SiteController.php(115): CController->render('login', Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#15 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#16 C:\xampp\htdocs\Exportc-Corp\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/Exportc-Corp/index.php?r=site/login
---
2014/12/04 19:07:29 [error] [exception.CException] exception 'CException' with message 'CAssetManager.basePath "C:\xampp\htdocs\Exportc-Corp\assets" is invalid. Please make sure the directory exists and is writable by the Web server process.' in C:\xampp\htdocs\yii\framework\web\CAssetManager.php:138
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CAssetManager.php(123): CAssetManager->setBasePath('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\yii\framework\web\CAssetManager.php(217): CAssetManager->getBasePath()
#2 C:\xampp\htdocs\yii\framework\gii\GiiModule.php(143): CAssetManager->publish('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\yii\framework\base\CComponent.php(111): GiiModule->getAssetsUrl()
#4 C:\xampp\htdocs\yii\framework\base\CModule.php(105): CComponent->__get('assetsUrl')
#5 C:\xampp\htdocs\yii\framework\gii\views\layouts\main.php(5): CModule->__get('assetsUrl')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#8 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#9 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div class="con...')
#10 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div class="con...')
#11 C:\xampp\htdocs\yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#12 C:\xampp\htdocs\yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#13 C:\xampp\htdocs\yii\framework\gii\views\layouts\column1.php(7): CBaseController->endContent()
#14 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#15 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#16 C:\xampp\htdocs\yii\framework\web\CController.php(784): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#17 C:\xampp\htdocs\yii\framework\gii\controllers\DefaultController.php(47): CController->render('login', Array)
#18 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DefaultController->actionLogin()
#19 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#20 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#21 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#22 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#23 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/default/log...')
#24 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#25 C:\xampp\htdocs\Exportc-Corp\index.php(13): CApplication->run()
#26 {main}
REQUEST_URI=/Exportc-Corp/index.php?r=gii/default/login
---
2014/12/06 18:21:26 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in C:\xampp\htdocs\yii\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(450): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(634): CApplication->getDb()
#5 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(667): CActiveRecord->getDbConnection()
#6 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1455): CActiveRecord->getCommandBuilder()
#7 C:\xampp\htdocs\Exportc-Corp\protected\controllers\SiteController.php(104): CActiveRecord->find('Usuario=? AND C...', Array)
#8 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\Exportc-Corp\index.php(16): CApplication->run()
#16 {main}
REQUEST_URI=/Exportc-Corp/index.php?r=site/login
HTTP_REFERER=http://localhost:8080/Exportc-Corp/
---
2014/12/06 18:21:30 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in C:\xampp\htdocs\yii\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(450): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(634): CApplication->getDb()
#5 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(667): CActiveRecord->getDbConnection()
#6 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1455): CActiveRecord->getCommandBuilder()
#7 C:\xampp\htdocs\Exportc-Corp\protected\controllers\SiteController.php(104): CActiveRecord->find('Usuario=? AND C...', Array)
#8 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\Exportc-Corp\index.php(16): CApplication->run()
#16 {main}
REQUEST_URI=/Exportc-Corp/index.php?r=site/login
HTTP_REFERER=http://localhost:8080/Exportc-Corp/
---
2014/12/06 18:21:31 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in C:\xampp\htdocs\yii\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(450): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(634): CApplication->getDb()
#5 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(667): CActiveRecord->getDbConnection()
#6 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1455): CActiveRecord->getCommandBuilder()
#7 C:\xampp\htdocs\Exportc-Corp\protected\controllers\SiteController.php(104): CActiveRecord->find('Usuario=? AND C...', Array)
#8 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\Exportc-Corp\index.php(16): CApplication->run()
#16 {main}
REQUEST_URI=/Exportc-Corp/index.php?r=site/login
HTTP_REFERER=http://localhost:8080/Exportc-Corp/
---
2014/12/06 18:23:20 [error] [exception.CException] exception 'CException' with message 'Property "Perfil.ultimoIngreso" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('ultimoIngreso', '2014-12-06 18:2...')
#1 C:\xampp\htdocs\Exportc-Corp\protected\controllers\SiteController.php(106): CActiveRecord->__set('ultimoIngreso', '2014-12-06 18:2...')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 C:\xampp\htdocs\Exportc-Corp\index.php(16): CApplication->run()
#10 {main}
REQUEST_URI=/Exportc-Corp/index.php?r=site/login
HTTP_REFERER=http://localhost:8080/Exportc-Corp/
---
2014/12/06 18:23:27 [error] [exception.CException] exception 'CException' with message 'Property "Perfil.ultimoIngreso" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('ultimoIngreso', '2014-12-06 18:2...')
#1 C:\xampp\htdocs\Exportc-Corp\protected\controllers\SiteController.php(106): CActiveRecord->__set('ultimoIngreso', '2014-12-06 18:2...')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 C:\xampp\htdocs\Exportc-Corp\index.php(16): CApplication->run()
#10 {main}
REQUEST_URI=/Exportc-Corp/index.php?r=site/login
HTTP_REFERER=http://localhost:8080/Exportc-Corp/
---
2014/12/06 18:24:29 [error] [exception.CException] exception 'CException' with message 'Property "Perfil.ultimoIngreso" is not defined.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('ultimoIngreso', '2014-12-06 18:2...')
#1 C:\xampp\htdocs\Exportc-Corp\protected\controllers\SiteController.php(106): CActiveRecord->__set('ultimoIngreso', '2014-12-06 18:2...')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 C:\xampp\htdocs\Exportc-Corp\index.php(16): CApplication->run()
#10 {main}
REQUEST_URI=/Exportc-Corp/index.php?r=site/login
HTTP_REFERER=http://localhost:8080/Exportc-Corp/
---
2014/12/06 21:28:23 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "/site/servicios".' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\Exportc-Corp\protected\views\layouts\main.php(161): CController->renderPartial('/site/servicios')
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#5 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#8 C:\xampp\htdocs\Exportc-Corp\protected\views\layouts\column1.php(6): CBaseController->endContent()
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#10 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(784): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#12 C:\xampp\htdocs\Exportc-Corp\protected\controllers\SiteController.php(44): CController->render('index')
#13 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#19 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#20 C:\xampp\htdocs\Exportc-Corp\index.php(16): CApplication->run()
#21 {main}
REQUEST_URI=/Exportc-Corp/
---
2014/12/06 21:28:27 [error] [exception.CException] exception 'CException' with message 'SiteController cannot find the requested view "/site/servicios".' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\Exportc-Corp\protected\views\layouts\main.php(161): CController->renderPartial('/site/servicios')
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#5 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#8 C:\xampp\htdocs\Exportc-Corp\protected\views\layouts\column1.php(6): CBaseController->endContent()
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#10 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(784): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#12 C:\xampp\htdocs\Exportc-Corp\protected\controllers\SiteController.php(44): CController->render('index')
#13 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#19 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#20 C:\xampp\htdocs\Exportc-Corp\index.php(16): CApplication->run()
#21 {main}
REQUEST_URI=/Exportc-Corp/index.php
HTTP_REFERER=http://localhost:8080/Exportc-Corp/
---
2014/12/07 01:54:29 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in C:\xampp\htdocs\yii\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(450): CModule->getComponent('db')
#4 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(634): CApplication->getDb()
#5 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(667): CActiveRecord->getDbConnection()
#6 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1455): CActiveRecord->getCommandBuilder()
#7 C:\xampp\htdocs\Exportc-Corp\protected\controllers\SiteController.php(104): CActiveRecord->find('Usuario=? AND C...', Array)
#8 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 C:\xampp\htdocs\Exportc-Corp\index.php(16): CApplication->run()
#16 {main}
REQUEST_URI=/Exportc-Corp/index.php?r=site/login
HTTP_REFERER=http://localhost:8080/Exportc-Corp/
---
